src/components/BuildStatus/BuildStatus.tsx

```diff
@@ -37,11 +41,11 @@ export const BuildStatus = ({ build, long, includeBuildLink, includeConfigLink }
     dateTitle = 'Submit Time';
   }
   return (
-    <div className="build-status">
+    <span className="build-status">
       <BuildStatusIcon build={build} />
       <BuildName build={build} includeBuildLink={includeBuildLink} includeConfigLink={includeConfigLink} long={long} />
-      <span title={dateTitle}>{createDateTime({ date: dateString! }).custom}</span>
-      <b>{build.user?.username && <Username text={build.user.username} />}</b>
-    </div>
+      {!hideDatetime && <span title={dateTitle}>{createDateTime({ date: dateString! }).custom}</span>}
+      {!hideUsername && <b>{build.user?.username && <Username text={build.user.username} />}</b>}
```

Comment:
same with above, maybe we should use "show..." instead of "hide..."? and set it to false as default if necessary.

Reply:
The same as above.